index.html

```diff
@@ -71,6 +71,51 @@
         This document is a work in progress.
       </p>
     </section>
+    <section class="informative">
+      <h2>
+        Introduction
+      </h2>
+      <p>
+        The Screen Orientation API allows web applications that would be
+        negatively impacted by users changing the orientation of their device
+        to block any changes. It also provides the current screen orientation
+        and any orientation changes so all web applications can display
+        correctly.
+      </p>
+      <p>
+        This specification describes how the API:
+      </p>
+      <ul>
+        <li>provides the current screen orientation type and angle of a device
+        as defined in the <a>updating orientation algorithm</a>.
+        </li>
+        <li>updates the screen orientation when the viewport angle changes
+        according to the <a>screen orientation change algorithm</a>.
+        </li>
+        <li>locks the screen to a specific orientation following the <a>locking
+        orientation algorithm</a>.
+        </li>
+        <li>unlocks the locked screen to the default orientation as described
+        in the <a data-link-for="ScreenOrientation">unlock()</a> method.
+        </li>
+      </ul>
+      <section>
+        <h2>
+          Goals
+        </h2>
+        <ul>
+          <li>Allow a web application to have control over screen orientation
```

Comment:
We probably want to say something about "without being intrusive to end users". Like, a developer can't randomly change screen orientation whenever they want... only when the security conditions are met. That's the goal :)